nodejs/eks/cluster.ts

```diff
@@ -928,11 +931,9 @@ export class Cluster extends pulumi.ComponentResource {
         return new NodeGroup(name, {
             ...args,
             cluster: this.core,
-            nodeSecurityGroup: this.nodeSecurityGroup,
-            clusterIngressRule: this.eksClusterIngressRule,
```

Comment:
Why is it correct to remove these?  I can't quite follow the refactorings around these two.

Reply:
🤦‍♂ i meant to re-tie this back to `this.core` and totally forgot as I was refactoring. I've updated the PR. Thank you once again for the 👁's